db_functions/partitioned_tables_manage_trigger_function.sql

```diff
@@ -26,40 +26,52 @@ DECLARE
 BEGIN
     IF ( TG_OP = 'DELETE' )
     THEN
-        alter_stmt = 'ALTER TABLE ' ||
-                     quote_ident(OLD.schema_name) || '.' || quote_ident(OLD.partition_of_table_name) ||
-                     ' DETACH PARTITION ' ||
-                     quote_ident(OLD.schema_name) || '.' || quote_ident(OLD.table_name) ||
-                     ' ;';
+        IF ( OLD.active )
```

Comment:
The multiple `active` checks here prevent unnecessary (and unwanted) action when updating the partitiion_parameters when the record is inactive.